### PR TITLE
Delete

### PR DESCRIPTION
**Pull request by CasparVelsen**
Added a delete function for Cards and Catches

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -48,6 +51,22 @@ export default function App() {
       body: JSON.stringify(formData),
     });
   }
+
+  async function handleDeleteCard(_id) {
+    console.log(_id);
+    const filteredCards = cards.filter(card => card._id !== _id);
+    setCards(filteredCards);
+
+    console.log(JSON.stringify({ _id }));
```
Comment (Sven-Lyco) [inline src/App.js:60]: 👀

File: src/App.js [hunk 2]
```diff
@@ -48,6 +51,22 @@ export default function App() {
       body: JSON.stringify(formData),
     });
   }
+
+  async function handleDeleteCard(_id) {
+    console.log(_id);
```
Comment (Sven-Lyco) [inline src/App.js:56]: 👀

File: src/App.js [hunk 3]
```diff
@@ -24,7 +24,10 @@ export default function App() {
   return (
     <>
       <Routes>
-        <Route path="/" element={<HomePage cards={cards} />} />
+        <Route
+          path="/"
+          element={<HomePage cards={cards} handleDelete={handleDeleteCard} />}
```
Comment (Sven-Lyco) [inline src/App.js:29]: Maybe you can choose the same name for the prop? :) 
`handleDeleteCard={handleDeleteCard}`

**Review by Sven-Lyco — COMMENTED** [review]
Hey Caspar, you did a good here and the code looks good to me :) 
Just a few comments 🚀

File: src/App.js (hunk 2 above)
Comment (CasparVelsen) [reply src/App.js:56]: 👍

File: src/App.js (hunk 1 above)
Comment (CasparVelsen) [reply src/App.js:60]: 😂

**Pull request merged**